### ISSUE TITLE
Allow multiple SQL queries

### ISSUE THREAD

**Issue by OndraZizka** [OndraZizka/csv-cruncher#9]
Something like

    -sql-output1="SELECT ..." -sql-output2="SELECT ..."

Or rather:
    
    crunch ...
    -sql="SELECT ..." -out foo.csv - sql="SELECT ..." -out bar.csv

**Comment by OndraZizka** [comment]
Depends on #53

**Comment by OndraZizka** [comment]
Closing in favor of #55
